courses/views.py

```diff
@@ -106,3 +116,52 @@ def user_existence(request):
         return Response({"error": "Must provide a UID"}, status=400)
 
     return Response({"exists": EdxAuthor.objects.filter(edx_uid=uid).exists()})
+
+
+@api_view(['POST'])
+def create_ccx(request):
+    """
+    Supports creating a CCX on edx.
+    """
+    def upstream_error(msg=None):
+        """Generate logged errors for upstream response errors"""
+        err = {
+            'error': 'Unable to create course on upstream edx server. {}'.format(msg)
+        }
+        log.error(err['error'])
+        return Response(err, status=502)
+
+    for required in ('master_course_id', 'user_email', 'total_seats', 'display_name'):
+        if required not in request.POST:
+            return Response({
+                'error': 'You did not supply required POST argument {}'.format(required),
+            }, status=400)
```

Comment:
why not check all of them and return an unique error with all the required fields?